unit/java_bytecode/java_bytecode_parser/lambda_examples/readme.md

```diff
@@ -0,0 +1,19 @@
+Each sub directory inside this one contains class files which have been compiled with specific java compilers. The proceedure taken to produce these files is as follows -
+ 1. Install the java compilers of interest. Skip this step if you already have them installed.
+ 2. Use `sudo update-alternatives --config javac` to configure your system to use the compiler of interest.
+ 3. Use `javac -version` to confirm the desired compilier has been correctly configured. The version information for each compiler is as follows -
+    * openJdk 8     - 1.8.0_151
+    * oracle java 8 - 1.8.0_161
+    * oracle java 9 - 9.0.4
+ 4. Compile the `.java` files using `javac -g ./*.java`. The `-g` flag switches on debugging output, which we require.
```

Comment:
You can use `-d` on javac to specify the output directory, but it requires it to exist first, so don't know if it actually makes it much easier to follow the instructions...